test-interface/src/main/scala/org/scalajs/testinterface/HTMLRunner.scala

```diff
@@ -147,6 +165,10 @@ protected[testinterface] object HTMLRunner extends js.JSApp {
       updateCounts()
     }
 
+    def warnNewTestsAdded(): Unit = {
+      rootBox.log("Potentially excluding new tests.", "warn")
```

Comment:
Good point. I avoided the word "suite" though, since we use it in the interface to refer to a test "class".